src/core/styles/common/constants/black.ts

```diff
@@ -0,0 +1,3 @@
+export enum Black {
```

Comment:
General question: Do we need these constants defined? Aren't these supposed to be defined in the `appTheme`?

Reply:
Ahh that's right 😆 ! Totally forgot about that lol Thanks for catching this!

Yeah we will need to pass `theme` to `Emotion` and then use it for styling!

We can do it after Create-a-thon when we have time to review everything!